pkg/config/types.go

```diff
@@ -171,4 +177,6 @@ type Config struct {
 	EC2DescribeInstancesBurst int
 	//Dynamic File Path for DynamicFile BackendMode
 	DynamicFilePath string
+	//use UserId for mapping strictly
```

Comment:
Define what "strictly" means in the comment.  As in, does not support mapping by ARN.

Reply:
thank you. I have updated